### DISCUSSION TITLE
When running multiple instances in different scopes how to use the label-enable flag?

### DISCUSSION THREAD

**Discussion by erikthegamer1242** [containrrr/watchtower#1239] [Q&A] [answered]
So I run two instances of WatchTower and I would like to use the` --label=com.centurylinklabs.watchtower.enable=true `flag but with my specific scope

```
docker run --detach --name watchtower --restart=on-failure \
    --volume /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower \
    --label-enable --cleanup --interval 3600
docker run --detach --name archiveteam-warrior \
    --label=com.centurylinklabs.watchtower.enable=true \
    --restart=on-failure --publish 8001:8001 \
    atdr.meo.ws/archiveteam/warrior-dockerfile
```

the WatchTower and archiveteam warrior I would like to keep separate from everything else. How do I now define the scope and use it with enable flag?

**Comment by piksel** [comment, marked as answer]
You don't really need the enable flag, since watchtower will only touch containers that have the scope set to whatever scope watchtower was started with.
So replace the `com.centurylinklabs.watchtower.enable=true` label with `com.centurylinklabs.watchtower.scope=warrior-stuff` and run watchtower with `--scope warrior-stuff`:
```
docker run --detach --name watchtower --restart=on-failure \
    --volume /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower \
    --scope warrior-stuff --cleanup --interval 3600
docker run --detach --name archiveteam-warrior \
    --label=com.centurylinklabs.watchtower.scope=warrior-stuff \
    --restart=on-failure --publish 8001:8001 \
    atdr.meo.ws/archiveteam/warrior-dockerfile
```
